src/components/profiles-modals/ImportProfileModal.vue

```diff
@@ -144,6 +146,25 @@ export default class ImportProfileModal extends mixins(ProfilesMixin) {
         }
 
         this.profileImportFilePath = files[0];
+
+        if (this.profileMods.known.length === 0 || this.profileMods.unknown.length > 0) {
+            // Sometimes the reason some packages are unknown is that the mod is out of date,
+            // so let's try refreshing it
+            try {
+                this.activeStep = 'REFRESH_MOD_LIST';
+                await this.$store.dispatch('tsMods/syncPackageList');
+                this.profileMods = await ProfileUtils.exportModsToCombos(
```

Comment:
Question: Is this needed? `syncPackageList` should update the mod list in Vuex, and that's watched by `updateProfileModsOnOnlineModListRefresh`, which does the same thing as this line? Seems to me we do this thing twice now?

Reply:
Good point, not needed